### DISCUSSION TITLE
How to custom the Windows title?

### DISCUSSION THREAD

**Discussion by mister-teddy** [opensumi/core#2137] [Q&A] [answered]
How can I custom the editor title like this: `Zalo Mini App Studio - ${Name of the App - not the folder name}`, or simply get rid of the `%5C` (I think it is the encoded blank space, it doesn't show like that on Mac and Linux) in the title?

![z3993538423378_2402349fc0b9bee8e9ed1d8219981a09](https://user-images.githubusercontent.com/29925961/209792638-ff3ff4b5-3907-4cef-9f3b-ac81faa25f93.jpg)

Thank you very much for such an amazing open source project 🙏

**Comment by yantze** [comment, marked as answer]
You can format the titlebar by getting an instance of IElectronHeaderService：
```javascript
injector.get(IElectronHeaderService).titleTemplate = '${activeEditorShort}${separator}${rootName}'
```
more info: https://github.com/opensumi/core/blob/742d745fc002eb773f3033739888930a5131fe2b/packages/electron-basic/src/common/header.ts#L23

**Reply by mister-teddy** [reply]
Thank you very much for the quick response, it's working now 🙏 

P/S: For those who couldn't import `import { IElectronHeaderService } from '@opensumi/ide-electron-basic/lib/common/header';`, you must upgrade @opensumi libs to version `^2.21.10`!
